src/components/Header.tsx

```diff
@@ -1,14 +1,22 @@
 import { useState } from "react";
 import { Dialog } from "@headlessui/react";
 import { Bars3Icon, XMarkIcon } from "@heroicons/react/24/outline";
+import { auth } from "../firebase";
+import { useAuthState } from "react-firebase-hooks/auth";
 
 const navigation = [
-  { name: "Families", href: "#" },
-  { name: "Wishes", href: "#" },
+  { name: "Families", href: "families" },
+  { name: "Wishes", href: "wishes" },
 ];
 
-export default function Header() {
+type HeaderProps = {
+  signInAction: () => void;
+  signOutAction: () => void;
+};
+
+export default function Header({ signInAction, signOutAction }: HeaderProps) {
   const [mobileMenuOpen, setMobileMenuOpen] = useState(false);
+  const [user] = useAuthState(auth);
```

Comment:
Not sure what the best practice is here. Re-fetching the user auth state or pass it down as a prop. They seem to have the same result